QuickBytes/src/main/java/com/quickbytes/controller/CustomerController.java

```diff
@@ -1,4 +1,4 @@
-package com.quickbytes.controller;
+package com.quickbytes.backend.controller;
```

Comment:
All the package names should be without the ".backend"

Reply:
Resolved